components/brave_rewards/resources/page/components/arriving_soon_message.tsx

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import { MoneyBagIcon } from '../../shared/components/icons/money_bag_icon'
+
+import { formatMessage } from '../../shared/lib/locale_context'
+import { getLocale } from '../../../../common/locale'
```

Comment:
You can use `LocaleContext` instead of `getLocale` since we're in a new-style component. See `manage_wallet_button.tsx` in this folder for an example.

Reply:
Awesome, thanks for the tip! Fixed.